dolphinscheduler-worker/src/test/java/org/apache/dolphinscheduler/server/worker/registry/WorkerStrategyTest.java

```diff
@@ -0,0 +1,159 @@
+package org.apache.dolphinscheduler.server.worker.registry;
```

Comment:
Please add `Apache 2.0` license header to every new file. CI blocks  because of missing license header.